README.md

```diff
@@ -87,9 +87,31 @@ These are the commands available to use with Action Scheduler:
 
 The best way to get a full list of commands and their available options is to use WP CLI itself. This can be done by running `wp action-scheduler` to list all Action Scheduler commands, or by including the `--help` flag with any of the individual commands. This will provide all relevant parameters and flags for the command.
 
-### Improving Performance with `--group`
+### Cautionary Note on Action Dependencies when using `--group` or `--hooks` Options
 
-Being able to run queues for specific groups of actions is valuable at scale. Why? Because it means you can restrict the concurrency for similar actions.
+The `--group` and `--hooks` options should be used with caution if you have an implicit dependency between scheduled actions based on their schedule.
+
+For example, consider two scheduled actions for the same subscription:
+
+* `scheduled_payment` scheduled for `2015-11-13 00:00:00` and
+* `scheduled_expiration` scheduled for `2015-11-13 00:01:00`.
+
+Under normal conditions, Action Scheduler will ensure the `scheduled_payment` action is run before the `scheduled_expiration` action. Becuase that's how they are scheduled.
+
+However, if using the `--hooks` option, and processing payment actions in a separate queue to expirations action, this dependency is not guaranteed.
```

Comment:
This sentence reads a bit awkwardly to me. I think it should be tweaked to something like this instead:

However, when using the `--hooks` option to process the `scheduled_payment` and `scheduled_expiration` actions separately, this dependency is not guaranteed.